src/components/Product.js

```diff
@@ -0,0 +1,22 @@
+//display name photo and description
+//display price formatted in dollars and cents 00.00
+//each product has an add to cart button. button text= Add to Cart
+
+import React from 'react'
+import './Products.css'
+import formatPrice from '../helpers/formatPrice'
+
+function Product(props) {
+    const {name, price, img, description, addToCart, item} = props
```

Comment:
Looks like you are trying to use a prop named `item` but you don't pass a prop named item when you create a Product inside of Products.js